### ISSUE TITLE
Plugins cannot check for updates

### ISSUE THREAD

**Issue by IIRobotNinjaII** [Strencher/BdBrowser#9]
On edge , on startup many plugins can't seem to check for updates and can't download additional libraries or new updates automatically

**Comment by huds0nx** [comment]
running into this as well. also getting errors that it can't download emotes and another error saying "Could not initiate BDFDB Library Plugin. Check your Internet Connection and make sure GitHub isn't blocked by your Network or try disabling your VPN/Proxy.".

**Comment by Strencher** [comment]
Yes, I got lots of reports about that, I can't reproduce that so that makes it harder to fix the issue. I believe it's some CSP (Content Secure Policy) issue or the browser doesn't like me remove it. It could also be a firewall issue.

**Comment by huds0nx** [comment]
that could definitely be a cause. just in case, here are a few screenshots of the errors i am getting:
<img width="467" alt="CleanShot 2021-10-05 at 16 17 10@2x" src="https://user-images.githubusercontent.com/39014019/136097032-cee51f64-e831-455e-af0d-9a642ae98636.png">
<img width="459" alt="CleanShot 2021-10-05 at 16 17 27@2x" src="https://user-images.githubusercontent.com/39014019/136097040-4c91036e-faae-4045-8e80-1fa13d452e92.png">

<img width="549" alt="CleanShot 2021-10-05 at 16 18 56@2x" src="https://user-images.githubusercontent.com/39014019/136097051-8c7a0e84-7072-48e2-8a2e-83b14c857814.png">
<img width="545" alt="CleanShot 2021-10-05 at 16 18 23@2x" src="https://user-images.githubusercontent.com/39014019/136097064-8a177237-bf0f-4d65-b2ed-f6fc9c41d004.png">

don't know if it would mean anything but i am using chrome on the latest version, on macos (specifically an m1 mac). also, where are the plugins and themes stored on the computer?

**Comment by huds0nx** [comment]
I know you told him to run the fetch command, but I just ran it too and got this
<img width="457" alt="CleanShot 2021-10-06 at 12 42 37@2x" src="https://user-images.githubusercontent.com/39014019/136247355-c22ec2ad-ea65-4006-855a-0a4eb134d46b.png">
:

**Comment by huds0nx** [comment]
okay so I just launched chrome with no cors (--disable-web-security), and I still got the error. I think this means it is not a problem with CORS when it gives the initializing bdfb and emojis
